### PR TITLE
Display User Details using mock service

### PR DESCRIPTION
**Pull request by RamuKammagani-osi**

### CODE REVIEW

File: app/javascript/sagas/getDetailsSaga.js [hunk 1]
```diff
@@ -0,0 +1,29 @@
+import UserService from '../_services/users';
+import * as actionTypes from '../actions/actionTypes';
+import { takeLatest, call, put } from 'redux-saga/effects';
+
+const id = 'id';
+// worker saga: makes the api call when watcher saga sees the action
+export function* getDetails() {
+  try {
+    const details = yield call(UserService.fetchUserDetails, id);
```
Comment (kelfink) [inline app/javascript/sagas/getDetailsSaga.js]: So this is a hard-coded 'fake' id for now?

File: app/javascript/selectors/detailSelector.js [hunk 2]
```diff
@@ -0,0 +1,5 @@
+export const selectDetailRecords = state => {
+  console.log('usersDetails state', state);
```
Comment (kelfink) [inline app/javascript/selectors/detailSelector.js]: OK for now if we're still just expecting mock service, but ultimately this needs to be removed.

File: app/javascript/views/userDetail/UserDetail.jsx [hunk 3]
```diff
@@ -5,74 +5,65 @@ import ShowField from '../../common/ShowField';
...
+const UserDetail = ({ details }) => (
   <div>
     <GlobalHeader profileName="County CWDS-Admin" profileId="profile.id" />
     <PageHeader pageTitle="Manage Users" button="" />
     <div className="container">
       <div className="row">
         <div className="col-md-12">
           <Cards
-            cardHeaderText="Manage Users: {county_name}"
+            cardHeaderText={'Manage Users: Sacramento'}
```
Comment (kelfink) [inline app/javascript/views/userDetail/UserDetail.jsx]: Do we have access to an object that represents our current user (logged in) so we can find the county / region?  Or do we need to get this back from the API ?

Comment (Sushmareddy-OSI) [reply app/javascript/views/userDetail/UserDetail.jsx]: +1

Comment (RamuKammagani-osi) [reply app/javascript/views/userDetail/UserDetail.jsx]: Sure! I'll update that, Thanks

File: app/javascript/selectors/detailSelector.test.js [hunk 4]
```diff
@@ -0,0 +1,35 @@
+import { selectDetailRecords } from './detailSelector';
+
+describe('selectors', () => {
+  describe('#selectUserRecords', () => {
+    it('selects the user records', () => {
```
Comment (Sushmareddy-OSI) [inline app/javascript/selectors/detailSelector.test.js]: change the description to select detail records

Comment (RamuKammagani-osi) [reply app/javascript/selectors/detailSelector.test.js]: Will do, thanks

**Pull request merged**